zipkin-storage/cassandra/src/main/resources/zipkin2-schema-upgrade-2.cql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS zipkin2.remote_service_by_service (
```

Comment:
@michaelsembwever @llinder @drolando can one of you have a look at this file? I'm just sort of replacing inline with what we did for span name indexing, but someone who knows cassandra should weigh in.

Reply:
It's an anti-pattern, you want to be careful with this, as it involves hot partitions. (That's why LCS with a low threshold and row caching. Deduping the writes is also important.)
So long as you apply the caution we took with `span_by_service` it's cool.

btw there's the stress profile if you want to see how the performance/throughput numbers differ…

Reply:
I will add comments to all the tables that have deduplication on them, as yeah in the schema now we make no such notes yet.

Reply:
added a commit revising comments, feel free to suggest revisions to those if needed. thanks!

Reply:
also backfilled missing stress tests